app/tag.js

```diff
@@ -1,6 +1,8 @@
 /* eslint no-template-curly-in-string: 0 eslint-comments/no-unlimited-disable: 1 */
 'use strict'
 ;(function (global, document, element, state) {
+  element = document.getElementById('${elementId}')
+  element.setAttribute('${attrName}', JSON.stringify({ status: 'Loading' }))
```

Comment:
where is this `state` use?

Reply:
the state has three statuses: `Loading` , `Blocked` and `Failed`

https://github.com/uncovertruth/userdive-tag-paster/pull/143/files#diff-378a5288126f16880a1d45efb628a424R59